spec/requests/tweets_spec.rb

```diff
@@ -0,0 +1,114 @@
+require 'rails_helper'
+
+RSpec.describe "Tweets API", type: :request do
+
+  describe "Tweet Creation" do
+      let(:user) { create(:user)}
+      let(:tweet) { create(:tweet, user: user)}
+      #tweet_params = (:tweet_body "holas")
+      it "creates a new tweet and returns a 200 status with matching JSON schema" do 
+      post "/tweets" #param {:tweet , :tweet_body: "hello", :user_id: user}
```

Comment:
You are leaving outside the parameters you need to sen to the controller, whtiout those parametters the controller would not to know how to create new tweets